skins/laika/src/components/pages/donation_form/PaymentSummary.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+	<div class="payment-summary has-outside-border has-margin-top-0 has-margin-bottom-18">
+			<p class="has-margin-bottom-18" v-html="getSummary()"></p>
+			<b-button @click="$emit( 'previous-page' )"
+					type="is-primary is-low"
+					outlined>
+				{{ $t('donation_form_section_back') }}
+			</b-button>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend( {
+	name: 'PaymentSummary',
+	props: [ 'amount', 'interval', 'paymentType' ],
+	methods: {
+		getSummary: function () {
+			// TODO find a way to create a computed proiperty instead of method where $t is still available
```

Comment:
Because in my understanding, methods in Vue components are either

* semi-private methods called from inside the component
* event handlers
* a public API that can be called from `$ref` and `$parent` references (although sparingly, I would rather use events for that).

This method returs a value that is then  used in the template. In my understanding, that's exactly what computed properies are for.

Reply:
The docs advise to use methods instead of computed properties in cases where you won't benefit from caching.
https://vuejs.org/v2/guide/computed.html#Computed-Caching-vs-Methods